src/components/end-user-contacts/end-user-contacts.tsx

```diff
@@ -61,6 +63,12 @@ const EndUserContacts = (): JSX.Element => {
       dispatch(setShowContactForm(false));
       newMsg.content = "";
       dispatch(sendMessagePreview(newMsg));
+
+      if(endUserContacts.comment) {
+        const commentMsg = getContactCommentNewMessage(endUserContacts.comment, chatId, contactMsgId, t);
+        dispatch(sendNewMessage(commentMsg));
```

Comment:
I think we need to figure another way to do this, please if you send a new message here then the bot will respond because the chat still belongs to the bot

Reply:
What if we introduced new flag variable to skip the bot when sending new message, something like [this](https://github.com/buerokratt/Buerokratt-Chatbot/pull/463/files#diff-35cefb9c7236976207c9a743ad1efabdfefc353c051b7898a810dad6a7a9dc56)?.

Reply:
Also, I've updated this PR to use the flag variable just in case.

Reply:
I think its a good solution